coalib/output/ConsoleInteraction.py

```diff
@@ -540,11 +543,11 @@ def get_action_info(section, action, failed_actions):
     """
     params = action.non_optional_params
 
-    for param_name in params:
+    for param_name in params:  # pragma: no cover
```

Comment:
bump what @yukiisbored said. #pragma no cover is used when that particular bit of code is really hard to test (say mockups for specific distributions, or things that require a folder hierarchy), but params like here can be mocked easily.
also, if you really insist its hard to be tested, at least add a reason in a comment